curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733c5e54e3a154c8078ed48.md

```diff
@@ -86,15 +119,31 @@ Unfortunately, we've just realized we read the requirements wrong. We need to al
 const regex = /^[a-zA-Z]{0,}\d{4,6}$/;
 ```
 
-But our pattern is getting long again. Thankfully, there's another short-hand for "match the previous character zero or more times" –  the asterisk symbol. Let's replace our quantifier with that in the pattern, and test it:
+But our pattern is getting long again. Thankfully, there's another short-hand for "match the previous character zero or more times" –  the asterisk (`*`) symbol. Let's replace our quantifier with that in the pattern, and test it:
```

Comment:
```suggestion
But our pattern is getting long again. Thankfully, there's another short-hand for "match the previous character zero or more times" –  the asterisk symbol (`*`). Let's replace our quantifier with that in the pattern, and test it:
```
More a question than a suggestion. Is it normal in English to place the actual symbol there? To me it sounds more natural to have it after the word `symbol`.

Reply:
I have seen it both ways. So it should be fine to leave it as is